stl/inc/algorithm

```diff
@@ -5585,6 +5591,22 @@ _CONSTEXPR20 void make_heap(_RanIt _First, _RanIt _Last) { // make [_First, _Las
 #ifdef __cpp_lib_concepts
 namespace ranges {
     // VARIABLE ranges::make_heap
+    // clang-format off
+    template <random_access_iterator _It, class _Pr, class _Pj>
+        requires sortable<_It, _Pr, _Pj>
+    static constexpr void _Make_heap_common(_It _First, _It _Last, _Pr _Pred, _Pj _Proj) {
+        // make nontrivial [_First, _Last) into a heap with respect to _Pred and _Proj
+        // clang-format on

```

Comment:
That line here is strange, usually we disable whole functions not parts of it

Reply:
Yes, I stumbled across this almost by accident but the intent is that it becomes the new convention. I would swear that I had tried this placement in the past and had poor results - maybe it was with a trailing-requires clause? Anyway, yes, as STL says I'm happy about anything that narrows the scope of code we must format manually.